packages/mui-base/src/Option/Option.tsx

```diff
@@ -25,18 +25,9 @@ const Option = React.forwardRef(function Option<
   OptionValue,
   RootComponentType extends React.ElementType,
 >(props: OptionProps<OptionValue, RootComponentType>, forwardedRef: React.ForwardedRef<Element>) {
-  const {
-    children,
-    component,
-    disabled = false,
-    label,
-    slotProps = {},
-    slots = {},
-    value,
-    ...other
-  } = props;
+  const { children, disabled = false, label, slotProps = {}, slots = {}, value, ...other } = props;
 
-  const Root = component || slots.root || 'li';
+  const Root = slots.root || 'li';
```

Comment:
```suggestion
  const Root = slots.root ?? 'li';
```

To match other components